packages/common/src/hooks/useAppBarRect/useAppBarRect.test.tsx

```diff
@@ -39,9 +40,18 @@ describe('useAppBarRect', () => {
     expect(state).toEqual(expected);
   });
 
+  const Component = () => {
+    const renderCounter = useRef(0);
+    renderCounter.current = renderCounter.current + 1;
+    useAppBarRect();
+
+    return <span>{renderCounter.current}</span>;
+  };
+
   it('Triggers a new state update when the window dimensions change', () => {
-    const { result } = renderHook(useAppBarRect);
+    const { getByText } = render(<Component />);
 
-    expect(result.all.length).toBe(2);
+    const hasRenderedTwice = getByText(/2/);
```

Comment:
one thing missing from the new `renderHook` is a history of renders. I've had to improvise.